spec/services/between_categories/creator_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'BetweenCategoriesTransactions::Creator', type: :feature do
+  context 'when valid' do
+    let(:user) { create(:user) }
+
+    let(:amount) { BigDecimal.new(Faker::Number.decimal(3, 2)) }
+    let(:amount_from) { BigDecimal.new(10_000) }
+    let(:init_category_to_amount) { BigDecimal.new(Faker::Number.decimal(3, 2)) }
+
+    let(:category_from) { create(:category, user: user, amount: amount_from) }
+    let(:category_to) { create(:category, user: user, amount: init_category_to_amount) }
+
+    let(:params) do
+      {amount: amount, user_id: user.id, category_from_id: category_from.id,
+       category_to_id: category_to.id}
+    end
+
+    it 'creates BetweenCategoriesTransaction' do
+      expect do
+        BetweenCategoriesTransactions::Creator.new(params).create
+      end.to change(BetweenCategoriesTransaction, :count).by(1)
+    end
+
+    it 'creates Transaction' do
+      expect do
+        BetweenCategoriesTransactions::Creator.new(params).create
+      end.to change(Transaction, :count).by(1)
+    end
+
+    before do
```

Comment:
Maybe it will be useful to move this `before` and steps after in a separate customer `context` ?

Reply:
@sinpai I think that separate `context` doesn't need, because It one action. 
Before here, because it only applies to the next check, Maybe It is worse.

Reply:
@marderer when you use `before` between `it` blocks it means that something is changed for user. If you change something then you would like to document that change. And for documenting|describing steps we use `context`